_posts/2017-06-21-Devoxx-pl.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+authors: [bart_blommaerts, jeff_mesens]
+title: "Devoxx Poland 2017"
+image: /img/2017-devoxx-pl/devoxx-poland.jpg
+tags: []
+category: Conference
+comments: true
+---
+# Devoxx Poland 2017
+_(This post is still in progress and will be updated on a regular base)_
+
+Krakow in the ICE Krakow Congress Centre.
+
+![ICE Krakow Congress Centre](/img/2017-devoxx-pl/venue-1.jpg){: .image height="420px" }
+![ICE Krakow Congress Centre foyer](/img/2017-devoxx-pl/venue-2.jpg){: .image height="420px" }
+
+We started off day 1 with the keynote in the absolutely phenomenal main room:
+
+![Main room 1](/img/2017-devoxx-pl/room-1.jpg){: .image .fit }
+
+## Keynote: Speed without Discipline: a Recipe for Disaster ([Venkat Subramaniam](https://twitter.com/@venkat_s))
+
+Venkat kicked off the keynote, talking about a paradigm shift, that is happening right now in software development:
+In the nineties, everybody was doing **imperative programming**, using objects to implement functionality.
+Nowadays, this style of software development is shifting towards a more **declarative approach**.
+In imperative programming, developers focus on both **what** they want to do and **how** they want to do it. 
+In declarative programming, on the other hand, developers focus on **what** they want to do and use tools and libraries to facilitate their goal.
```

Comment:
In declarative programming on the other hand, developers ...
(delete the first comma)